### PR TITLE
GB-1509

### PR DESCRIPTION
**Pull request by osman**

### CODE REVIEW

**Comment by osman** [comment]
@willwarren please see the documentation

File: src/main/java/com/groupbyinc/api/Query.java [hunk 1]
```diff
@@ -176,4 +180,57 @@ public Query setMatchStrategy(MatchStrategy matchStrategy) {
...
+
+    /**
+     * <code>
+     * Specify which navigations should be returned. If set, this overrides the Navigations defined
+     * in Command Center and only returns the navigations included. If this parameter is blank the
+     * Navigations in Command Center are returned. If a navigation is specified that does not exist,
+     * it will be ignored.
+     *
+     * JSON Reference:
+     * { "includeNavigations": [ "width", "brand", "height" ] }
```
Comment (codeallthethingz) [inline src/main/java/com/groupbyinc/api/Query.java]: . notation example

File: src/main/java/com/groupbyinc/api/Query.java [hunk 2]
```diff
@@ -176,4 +180,57 @@ public Query setMatchStrategy(MatchStrategy matchStrategy) {
...
+     * @return A list of navigations that will be excluded from the response.
+     */
+    public List<String> getExcludeNavigations() {
+        return excludeNavigations;
+    }
+
+    /**
+     * <code>
+     * Specify which navigations should not be returned. If set, this forces the response to
+     * exclude certain Navigations defined in Command Center . If this parameter is blank all
```
Comment (codeallthethingz) [inline src/main/java/com/groupbyinc/api/Query.java]: space .

File: src/main/java/com/groupbyinc/api/Query.java [hunk 3]
```diff
@@ -176,4 +180,57 @@ public Query setMatchStrategy(MatchStrategy matchStrategy) {
...
+     * Navigations in Command Center are returned. If a navigation name is specified that does
```
Comment (codeallthethingz) [inline src/main/java/com/groupbyinc/api/Query.java]: I don't think navigations should be capitalized.

File: src/main/java/com/groupbyinc/api/Query.java [hunk 4]
```diff
@@ -176,4 +180,57 @@ public Query setMatchStrategy(MatchStrategy matchStrategy) {
...
+     */
+    public List<String> getIncludeNavigations() {
+        return includeNavigations;
+    }
+
+    /**
+     * <code>
+     * Specify which navigations should be returned. If set, this overrides the Navigations defined
+     * in Command Center and only returns the navigations included. If this parameter is blank the
+     * Navigations in Command Center are returned. If a navigation is specified that does not exist,
```
Comment (codeallthethingz) [inline src/main/java/com/groupbyinc/api/Query.java]: Navigations -- > navigations

**Pull request merged**